src/components/container.js

```diff
@@ -1,38 +1,160 @@
 import React, { Component } from "react";
 import PropTypes from "prop-types";
+import { ReactComponent as CloseIcon} from "../assets/closeIconPage.svg";
+import { setHeaderOption } from "../redux/action-creators";
+
+import { Link } from "react-router-dom";
 
 const Container = class Container extends Component {
+
   static propTypes = {
     title: PropTypes.string.isRequired,
     children: PropTypes.array,
   };
+
+  state = {
+    showPopUp: true,
+  };
+
+  togglePopUp = () => {
+    this.setState(prevState => ({ showPopUp: !prevState.showPopUp }),);
+  };
+
+
   render() {
+    const { showPopUp } = this.state;
     const { title, children } = this.props;
-    const container = {
+
+    const popUpBox = {
       width: "100%",
-      minHeight: "100%",
+      height: "100%",
+      background: "black",
+      opacity: "0.5",
       position: "absolute",
-      backgroundColor: "#f9f9f9",
+      zIndex: "3",
+      pointerEvents: "auto",
+      top: "0",
     };
-    const content = {
-      margin: '15vh 15vw'
+
+    const container = {
+      position: "fixed",
+      zIndex: "4",
+
+      display:"flex",
+      justifyContent:"center",
+      alignItems: "center",
+
+      width: "50%",
+      maxHeight:"60%",
+
+      top:"20%",
+      left:"25%",
+      
+      backgroundColor: "#ffffff",
+      borderRadius: "8px",
+      flexDirection: "column",
+
+      pointerEvents: "auto",
+
+
     };
+
+    const titleContent = {
+      width: "100%",
+      paddingTop: "28px",
+      paddingBottom: "20px",
+      paddingLeft: "40px",
+
+    }
+
+    const childrenContent = {
+      width: "100%",
+      paddingLeft: "40px",
+      paddingRight: "40px",
+      paddingTop: "20px",
+      paddingBottom: "20px",
+      overflowY: "scroll",
+      webkitOverflowScrolling: "touch",   
+
+    }
+
     const titleBox = {
-      fontSize: "40px",
-      borderBottom: "6px dashed #f79640",
-      paddingBottom: "15px",
-      marginBottom: "22px",
+      display: "flex",
+      flexDirection: "row",
+      borderBottom: "1px solid #CCC",  
+      width:"100%",
+      fontSize: "20px",
     };
 
+
+    const closeIcon = {
+      display: "flex",
+      flexDirection: "column",
+      paddingTop: "26px",
+      paddingBottom: "20px",
+      paddingRight: "30px",
+
+      cursor: "pointer",
+    }
     return (
-      <div style={container}>
-        <div style={content}>
-          <h3 style={titleBox}>{title}</h3>
-          {children}
-        </div>
-      </div>
+
+      <div>
+        
+        {
+          this.state.showPopUp
```

Comment:
`showPopUp` and `setHeaderOption` are both obsolete now that the visibility of modal dialogs is controlled solely by the URL. All use of those can be removed from container.js and header.js, simplifying them a fair bit.